lib/posix/Kconfig

```diff
@@ -65,6 +65,14 @@ config MAX_TIMER_COUNT
 	help
 	  Mention maximum number of timers in POSIX compliant application.
 
+config TIMER_CREATE_WAIT
```

Comment:
any reason to not name this `TIMER_CREATE_TIMEOUT`?

"timeout" is more accurate description of this symbol than "wait."